README.md

```diff
@@ -435,7 +435,6 @@ Emoji | Aliases
 <img src="img-buildkite-64/centos.png" width="20" height="20" alt="centos"/> | `:centos:`
 <img src="img-buildkite-64/codecov.png" width="20" height="20" alt="codecov"/> | `:codecov:`
 <img src="img-buildkite-64/compute.png" width="20" height="20" alt="compute"/> | `:compute:`, `:gcompute:`
-<img src="img-buildkite-64/cucumber.png" width="20" height="20" alt="cucumber"/> | `:cucumber:`, `:cucumber-io:`, `:cucumber-open:`, `:cucumber-studio:`
```

Comment:
This image is not referenced in the `img-buildkite-64.json` file. I think it can be deleted.